Extension/src/LanguageServer/configurations.ts

```diff
@@ -549,49 +549,106 @@ export class CppProperties {
         }
     }
 
-    public handleConfigurationEditCommand(onSuccess: (document: vscode.TextDocument) => void): void {
+    private async SetPropertiesFile(): Promise<void> {
```

Comment:
style: `setPropertiesFile`, though I might name this `ensurePropertiesFile` since it doesn't always set it.

Reply:
changed to `ensurePropertiesFile()`